frontend/src/diagram/DiagramWebSocketContainerMachine.ts

```diff
@@ -76,6 +76,7 @@ export interface DiagramWebSocketContainerContext {
   subscribers: Subscriber[];
   message: string | null;
   selectedObjectId: string | null;
+  justOpened: boolean;
```

Comment:
Is this boolean useful?

Reply:
You have two new booleans `justOpened` and `firstOpen` and you never read the value of `justOpened` so I believe that 50% of this PR is not used

Reply:
Yes, that code remains from previous try